request.go

```diff
@@ -495,27 +498,26 @@ func handleTime(attribute interface{}, args []string, fieldValue reflect.Value)
 		return reflect.ValueOf(t), nil
 	}
 
-	var at int64
+	switch v.Interface().(type) {
```

Comment:
The original logic was kept, since we're still checking the type of the `attribute`, while the previous/following checks are done to the `fieldValue` kind.

I wonder if we should check the `fieldValue` kind here as well for consistency.